benches/crypto.rs

```diff
@@ -89,6 +89,17 @@ mod ed25519_benches {
                 .map(|key| key.public().clone())
                 .collect();
 
+            let aggregate_signature =
+                BLS12381AggregateSignature::aggregate(blst_signatures.clone()).unwrap();
```

Comment:
Note to self: does `aggregate` *really* needs to consume the individual signatures, I wonder?

Reply:
It doesn't seem like it. I've created a new PR, #90, to fix this